projects/public-search/src/app/api/ill-request-api.service.ts

```diff
@@ -64,9 +64,10 @@ export class IllRequestApiService extends BaseApi {
     page: number,
     itemsPerPage: number = 10,
     headers = BaseApi.reroJsonheaders,
-    sort: string = '-created'
+    sort: string = '-created',
+    filters?: {[key: string]: string}
   ): Observable<Record | Error> {
     const query = `patron.pid:${patronPid} AND -status:denied AND -status:closed AND -loan_status:ITEM_RETURNED`;
```

Comment:
Remove the additional filters in this query. They are not useful anymore because the returned data is now filtered directly in the backend.

```typescript
const query = `patron.pid:${patronPid}`
```